app/src/shared/components/DatePicker/index.jsx

```diff
@@ -51,8 +51,8 @@ class DatePicker extends React.Component<Props> {
                             <React.Fragment>
                                 <TextField
                                     type={isMobile ? 'date' : 'text'}
-                                    value={value === 'No stored events added yet' ?
-                                        'No stored events added yet' :
+                                    value={value === 'Select date' ?
```

Comment:
Can someone help abstract this bit of text to `en.po`? It's horrible to have it here, as the user of the component must use this exact same string, otherwise the component won't work. (see `userpages/components/StreamPage/Show/HistoryView/index.jsx`)